test/e2e/mock-response-data/snaps/update-snap-binary.ts

```diff
@@ -0,0 +1,186 @@
+import fs from 'fs';
```

Comment:
:information_source: this is the script for handling the logic of updating a snap package with `yarn update-snap-binary`